examples/each.js

```diff
@@ -0,0 +1,69 @@
+const StringHelper = require('../index')
+
+let helper = new StringHelper()
+let people = [
+  { name: 'pedro', sex: 'm', age: 30 },
+  { name: 'leticia', sex: 'f', age: 21 },
+  { name: 'pablo', sex: 'm', age: 20 }
+]
+let titles = ['ID', 'Name', 'Sex', 'Age']
+
+const rows = function (element, index) {
```

Comment:
This function makes up a single row (`tr`) each call, not all of them, it would make more sense to name it singular.